src/stash/mod.rs

```diff
@@ -399,6 +407,8 @@ impl<V> Stash<V> {
             self.next_free = i;
```

Comment:
Could you document the fact that a cleared stash operates identically to a new stash?